### PR TITLE
Implement Anemometer Logic

### PR DESCRIPTION
**Pull request by aidantd**
WHAT
- Create logic for interacting with adc channel
- Create logic for reading from anemometer 

WHY
- We will need to read the wind speeds from the anemometer and have the system record it

### CODE REVIEW

File: main/peripherals/sen0170.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    return readVoltage * 6;
+}
+
+// Reads in the recorded voltage from the anemometer and converts it to a wind speed
+uint32_t readSen0170(void) {
+    uint32_t readVoltage = 0;
+    readVoltage = readADC(ADC_SEN0170);
+
+    return calculateWindSpeed(readVoltage);
+}
```
Comment (aidantd) [inline main/peripherals/sen0170.c:25]: Add logic for converting from m/s to mpi/hr

File: main/peripherals/sen0170.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+// Declarations
+#define ADC_SEN0170 ADC1_CHANNEL_0
+
+// Calculates the wind speed based off the formula provided by the manufacturer
+static uint32_t calculateWindSpeed(uint32_t readVoltage) {
+    return readVoltage * 6;
+}
+
+// Reads in the recorded voltage from the anemometer and converts it to a wind speed
```
Comment (aidantd) [inline main/peripherals/sen0170.c]: Give a better comment for this and what unit it returns

File: main/peripherals/sen0170.c (hunk 1 above)
Comment (aidantd) [reply main/peripherals/sen0170.c:25]: has been implemented

File: main/peripherals/sen0170.c (hunk 2 above)
Comment (aidantd) [reply main/peripherals/sen0170.c]: Has been expanded on

**Review by Quietfawn — APPROVED** [review]
GREAT

**Pull request merged**